.github/workflows/release_master_fontpatcher-zip.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release font patcher from master
```

Comment:
Maybe we can find a better name here (also for the yaml file).

Esp this is not a 'release' but some snapshot.
Maybe `Create font-patcher snapshot` and `font-patcher-snap.yml`? Hmm :thinking: 
Maybe `Create font-patcher archive` and `font-patcher-zip.yml`?

All other workflows have the `.yml` suffix, I think we should stick with that.

Reply:
Of the two you suggested, I like

> Maybe Create font-patcher archive and font-patcher-zip.yml?

more. “Snap” gives me these uncomfortably weird Ubuntu tinglies